### PR TITLE
Update for libdaos.so.1 major API version bump

### PR DESCRIPTION
**Pull request by kccain**
And packaging files update from pacakaging repo, along with spec
packaging fixes for leap15 builds.

PR-repos: daos@PR-3935:15
PR-repos: mpich@PR-44

Signed-off-by: Kenneth Cain <kenneth.c.cain@intel.com>

### CODE REVIEW

**Comment by brianjmurrell** [comment]
Update the `packaging/` directory before going further.

File: mpi4py.spec [hunk 1]
```diff
@@ -10,13 +33,15 @@
 %endif
 
 %global with_python3 1
-%if (0%{?suse_version} >= 1500)
+%if (%{distro} == "leap15")
 %global python3_pkgversion 3
 %global _mpich_load \
- module load gnu-mpich; \
+ module_load mpich;
```
Comment (brianjmurrell) [inline mpi4py.spec:39]: You dropped a line continuator here:
```suggestion
 module_load mpich; \
```

**Review by brianjmurrell — CHANGES_REQUESTED** [review]

File: packaging/Dockerfile.centos.7 [hunk 2]
```diff
@@ -1,5 +1,5 @@
 #
-# Copyright 2018-2019, Intel Corporation
+# Copyright 2018-2020, Intel Corporation
```
Comment (brianjmurrell) [inline packaging/Dockerfile.centos.7:2]:
```suggestion
# Copyright 2018-2021, Intel Corporation
```

File: packaging/Dockerfile.mockbuild [hunk 3]
```diff
@@ -1,5 +1,5 @@
 #
-# Copyright 2018-2019, Intel Corporation
+# Copyright 2018-2020, Intel Corporation
```
Comment (brianjmurrell) [inline packaging/Dockerfile.mockbuild:2]:
```suggestion
# Copyright 2018-2021, Intel Corporation
```

File: packaging/Dockerfile.ubuntu.20.04 [hunk 4]
```diff
@@ -0,0 +1,39 @@
+#
+# Copyright 2019-2020, Intel Corporation
```
Comment (brianjmurrell) [inline packaging/Dockerfile.ubuntu.20.04:2]:
```suggestion
# Copyright 2019-2021, Intel Corporation
```

**Review by brianjmurrell — CHANGES_REQUESTED** [review]
This looks like another that doesn't need a rebuild.

**Comment by kccain** [comment]
This PR is not needed, closing.

**Pull request closed without merging**